src/views/check-your-answers.njk

```diff
@@ -28,6 +28,15 @@
 
 {% set jurisdiction = jurisdictionMap[props.data.limitedPartnership.data.jurisdiction] %}
 
+{% set capitalContributionSubTypesMap = {
+  "MONEY": i18n.capitalContribution.contributionSubTypes.money,
```

Comment:
So here, just reference `i18n.capitalContribution.money` and not duplicate the values.

Reply:
As above